azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -348,8 +358,9 @@ private void registerCustomCredential(final Props props, final Credentials hadoo
   public KeyStore getKeyStore(final Props props) {
```

Comment:
Nit: Add comment for this method

Reply:
I was planning to address it in the other PR but will do it here for this one.